llvm/lib/IR/Attributes.cpp

```diff
@@ -616,6 +661,23 @@ std::string Attribute::getAsString(bool InAttrGrp) const {
     return Result;
   }
 
+  if (hasAttribute(Attribute::Initializes)) {
+    std::string Result;
+    raw_string_ostream OS(Result);
+    ConstantRangeList CRL = getValueAsConstantRangeList();
+    OS << "initializes(";
+    size_t i = 0;
+    for (auto &CR : CRL) {
```

Comment:
use the recently introduced `interleaveComma`

Reply:
Nice util! Applied `interleave` instead of `interleaveComma` to avoid the space after comma.